pkg/operator/controller/garden/reconciler_reconcile.go

```diff
@@ -224,7 +224,12 @@ func (r *Reconciler) reconcile(
 		renewVirtualClusterAccess = g.Add(flow.Task{
 			Name: "Renewing virtual garden access secrets after creation of new ServiceAccount signing key",
```

Comment:
It might make sense if `gardener-operator` would set the `gardener.cloud/operation=renew-garden-access-secrets` annotation to all seeds in this case too; not in the same task but in a new one with the same condition.

Reply:
Definitely! I see, gardener-operator doesn't manage the gardener control plane yet. I guess, we will add logic for renewing the seed credentials later on.
Should I add a corresponding task to https://github.com/gardener/gardener/issues/7016 for this?

Reply:
Done :)